pkg/crc/api/client/types.go

```diff
@@ -19,14 +19,16 @@ type StartResult struct {
 }
 
 type ClusterStatusResult struct {
-	CrcStatus        string
-	OpenshiftStatus  string
-	OpenshiftVersion string `json:"OpenshiftVersion,omitempty"`
-	DiskUse          int64
-	DiskSize         int64
-	RAMUse           int64
-	RAMSize          int64
-	Preset           preset.Preset
+	CrcStatus            string
+	OpenshiftStatus      string
+	OpenshiftVersion     string `json:"OpenshiftVersion,omitempty"`
+	DiskUse              int64
+	DiskSize             int64
+	RAMUse               int64
+	RAMSize              int64
+	PersistentVolumeUse  int
+	PersistentVolumeSize int
```

Comment:
doesn't that mean it will be required field for openshift preset also which is not the case right? So may be use `json:"PersistentVolumeUse/Size,omitempty"` for this also

Reply:
Done